clients/privacy-center/public/fides-js-components-demo.html

```diff
@@ -138,6 +138,9 @@
     <style>
       body {
         margin: 16px;
+        height: 100vh;
+        overflow: scroll;
+        overflow-x: hidden;
```

Comment:
Are you relying on these for mobile responsiveness/etc? Because this is just in the test page

Reply:
no, there was some weird behavior with the test page where scrolling wasn't working. But mobile responsiveness was fixed by removing this line- https://github.com/ethyca/fides/pull/4306/files#diff-e542dc6a9b835392e37fd0de6075230a5ca4a81daebe2271476e7c5ec9ada8fdL324

Reply:
Gotcha, thanks!